closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -63,10 +65,14 @@ compile-libs :
 	export \
 			$(OPENJCEPLUS_VS_LIB) \
 			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
+			JAVA_HOME=$(OPENJCEPLUS_JDK) \
 			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
 			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
-		&& $(MAKE) -j1 -C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) -f $(OPENJCEPLUS_JGSKIT_MAKE) all
+		&& $(MAKE) \
```

Comment:
Please confirm the intentional removal of `-j1` (the OpenJCEPlus makefiles are ready for parallel execution)?

Reply:
Yes, I updated the prerequisites for the targets to allow for parallel execution.